doc/acceptance.md

```diff
@@ -0,0 +1,22 @@
+#Acceptance test 
+How to run the acceptance tests.
+
+## Preconditions
+
+### What you need on your computer to run the test
+- Retrieve the content of the __acceptance folder
+- Install all the requirements listed by the __acceptance/requirements.txt__ file: `pip install -r acceptance/requirements.txt` (we advise you to use a dedicated environment. Take a look at [virtualenv](https://pypi.python.org/pypi/virtualenv) for that).
```

Comment:
Which python version is needed ?

Related to virtualenv, is this guide more relevant http://docs.python-guide.org/en/latest/dev/virtualenvs/ ?

I'm trying on Ubuntu

```
$ sudo apt-get install virtualenv
$ virtualenv -p /usr/bin/python2.7 autosleepenv
$ source autosleepenv/bin/activate
```

I'll feedback